Realm/RLMArray.h

```diff
@@ -26,59 +26,59 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
 
- RLMArray is the container type in Realm used to define to-many relationships.
+ `RLMArray` is the container type in Realm used to define to-many relationships.
 
- Unlike an NSArray, RLMArrays hold a single type, specified by the `objectClassName` property.
+ Unlike an `NSArray`, `RLMArray`s hold a single type, specified by the `objectClassName` property.
  This is referred to in these docs as the “type” of the array.
 
- When declaring an RLMArray property, the type must be marked as conforming to a
+ When declaring an `RLMArray` property, the type must be marked as conforming to a
  protocol by the same name as the objects it should contain (see the
- `RLM_ARRAY_TYPE` macro). RLMArray properties can also use Objective-C generics
+ `RLM_ARRAY_TYPE` macro). `RLMArray` properties can also use Objective-C generics
  if available. For example:
```

Comment:
"if available" is no longer applicable since we dropped support for Xcode 6. Probably a bunch of other similar little references to generics not being available.